src/redis.go

```diff
@@ -24,7 +25,9 @@ const (
 var args argumentList
 
 func main() {
-	integration, err := sdk.NewIntegration(integrationName, integrationVersion, &args)
+	integration, err := integration.New(integrationName, integrationVersion, integration.Args(&args))
+	fatalIfErr(err)
+	entity, err := integration.Entity(integrationName, args.Hostname)
```

Comment:
for this upgrading we need a feature flag to allow local entities for the old users.